### ISSUE TITLE
Add GDAL input for DEM files

### ISSUE THREAD

**Issue by der-stefan** [hoche/splat#26]
I'm still wondering if the SPLAT data files (*.sdf) are really necessary. Wouldn't it be possible to load a digital elevation model (DEM) into the RAM just after calculating the geographical extent?
In any case, it would be very useful to directly read geotiffs or other DEM file formats supported by GDAL directly without invoking srtm2sdf. For OpenTopoMap I prefer to have a global single geotif file, that is internally tiled. Selection of some extents is very fast.

**Comment by dBitech** [comment]
+1 for this, 
this would be the first step in allowing other DEM resolutions (think fractional meter LIDAR), available in many parts of the world to be used.

**Comment by der-stefan** [comment]
Yes, but just one step. If I remember correctly, the two selectable resolutions are hard-coded. But shouldn't be a problem to just use the resolution of the input DEM file.

**Comment by dBitech** [comment]
Correct, getting the pixel size (resolution) out of a gdal supported input DEM is trivial. The following explains the nuances using gdalinfo https://gis.stackexchange.com/questions/281132/why-doesnt-gdalinfo-report-pixel-size
